src/Microsoft.Diagnostics.Monitoring.EventPipe/Counters/CounterPayload.cs

```diff
@@ -73,43 +53,88 @@ internal class CounterPayload : ICounterPayload
         public string Metadata { get; }
 
         public EventType EventType { get; set; }
+
+        public virtual bool IsMeter => false;
+
+        public int Series { get; }
+    }
+
+    internal sealed class StandardCounterPayload : CounterPayload
```

Comment:
I'm wondering if StandardCounter is not the right name here.

Reply:
Switched this to EventCounterPayload - this seemed more descriptive, but this can be changed to something else.